architecture/core_concepts/routes.adoc

```diff
@@ -75,7 +75,7 @@ ifdef::openshift-enterprise,openshift-origin[]
 xref:../../admin_guide/high_availability.adoc#configuring-a-highly-available-service[High Availability]
 endif::[]
 ifdef::openshift-dedicated[]
-the link:https://docs.openshift.com/enterprise/3.1/admin_guide/high_availability.html#configuring-a-highly-available-service[{product-title} Enterprise Cluster Administration documentation]
+the link:https://docs.openshift.com/enterprise/3.1/admin_guide/high_availability.adoc#configuring-a-highly-available-service[{product-title} Enterprise Cluster Administration documentation]
```

Comment:
Since this is pointing to the OpenShift docs repo, it needs to be an "xref" to the file and not a "link":

xref:../admin_guide/high_availability.adoc#configuring-a-highly-available-service[{product-title} Enterprise Cluster Administration documentation]